src/sagas/filters.js

```diff
@@ -0,0 +1,165 @@
+import { takeEvery, select, put } from 'redux-saga/effects';
```

Comment:
Please write some comments near the top part of the file with:

- Logic we're implementing in terms of user stories
- Short explanation on what each action means
- Some implementation details where necessary for future us